scripts/run_celery.ps1

```diff
@@ -0,0 +1,3 @@
+$ENV:FORKED_BY_MULTIPROCESSING=1
+
+celery --app run_celery worker --pidfile="$env:TEMP\celery.pid" --pool=solo --loglevel=DEBUG --concurrency=4 -Q database-tasks,job-tasks,notify-internal-tasks,periodic-tasks,priority-tasks,bulk-tasks,reporting-tasks,research-mode-tasks,retry-tasks,send-sms-tasks,send-email-tasks,service-callbacks,delivery-receipts
```

Comment:
My Windows compatible celery run script. Ideally, I'd want to have some [shelljs instead for our scripts](https://github.com/shelljs/shelljs).